src/da65/handler.c

```diff
@@ -719,33 +736,38 @@ void OH_DirectImmediate (const OpcDesc* D)
 
 
 
+/* <bit> zp */
+/* NOTE: currently <bit> is part of the instruction */
 void OH_ZeroPageBit (const OpcDesc* D)
 {
-    unsigned Bit = GetCodeByte (PC) >> 5;
+    /* unsigned Bit = GetCodeByte (PC) >> 5; */
     unsigned Addr = GetCodeByte (PC+1);
 
     /* Generate a label in pass 1 */
     GenerateLabel (D->Flags, Addr);
 
     /* Output the line */
-    OneLine (D, "%01X,%s", Bit, GetAddrArg (D->Flags, Addr));
+    OneLine (D, "%s", GetAddrArg (D->Flags, Addr));
 }
 
 
 
+/* <bit> A */
+/* NOTE: currently <bit> is part of the instruction */
 void OH_AccumulatorBit (const OpcDesc* D)
 {
-    unsigned Bit = GetCodeByte (PC) >> 5;
+    /* unsigned Bit = GetCodeByte (PC) >> 5; */
```

Comment:
I personally - actually strongly - dislike commented-out code. Especially without a comment why it was commented-out. If there's nothing to say in such a comment, then the code should just be removed and it should be relied upon that the Git history provides better context/knowledge.